getting-started/robot.py

```diff
@@ -21,21 +25,22 @@ def autonomousInit(self):
 
     def autonomousPeriodic(self):
         """This function is called periodically during autonomous."""
-        
+
         # Check if we've completed 100 loops (approximately 2 seconds)
         if self.auto_loop_counter < 100:
-            self.robot_drive.drive(-0.5, 0) # Drive forwards at half speed
+            self.drive.arcadeDrive(-0.5, 0)  # Drive forwards at half speed
             self.auto_loop_counter += 1
         else:
-            self.robot_drive.drive(0, 0)    #Stop robot
+            self.drive.arcadeDrive(0, 0)  # Stop robot
 
     def teleopPeriodic(self):
         """This function is called periodically during operator control."""
-        self.robot_drive.arcadeDrive(self.stick)
+        self.drive.arcadeDrive(self.stick.getY(), self.stick.getX())
 
     def testPeriodic(self):
         """This function is called periodically during test mode."""
-        wpilib.LiveWindow.run()
+        pass
+
```

Comment:
Hm. We don't have a `robotPeriodic`, `teleopInit`, nor `testInit`, so why not just delete this unnecessary method?

Reply:
I left this in to mirror the allwpilib example linked in the first post.

Reply:
You linked the C++ gearsbot example. We're talking about the getting started example…

If you really wanted to mirror the getting started ([C++](https://github.com/wpilibsuite/allwpilib/blob/v2018.1.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/src/Robot.cpp) and [Java](https://github.com/wpilibsuite/allwpilib/blob/v2018.1.1/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gettingstarted/Robot.java), but you really should be looking at the Java one as we mirror WPILibJ, not WPILibC) example, you'd add in more stubs as well.

Reply:
Speaking of which, we should probably change the autonomous counter thing to use Timer instead, like in https://github.com/wpilibsuite/EclipsePlugins/commit/f4131c91ac01cd21bac36c8372d0ab7046d3c4b2#diff-c0b3b6a7b702a9920b95516dac2c0c7c.